src/main.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+/** 
+     * Реализовать декоратор debounce
+     * 
+     * Чтобы оригинальная функция запускалась только после определённой паузы в запусках обёртки
+     * Пока пользователь печататет - ничего
+     * Если остановился и подождал секунду - вывести последнее значение
+     * Функция onChange должна получать тот же `this` и аргументы, что и обёртка
+     **/
+const input = document.querySelector('input');
+let timer = null;
```

Comment:
It will be better if you put it in `debounce`

Reply:
Done